enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -511,6 +556,30 @@ class TestAdminPolicyCreateView(CRUDViewTestMixin, APITestWithMocks):
     ``SubsidyAccessPolicyViewSet`` implementation.
     """
 
+    def setUp(self):
+        super().setUp()
+        self.setup_mocks()
+
+    def setup_mocks(self):
```

Comment:
Code is identical to TestAuthenticatedPolicyCRUDViews.setup_mocks, can we break it out?